src/main/java/com/example/myshoppingapp/model/BaseEntity.java

```diff
@@ -0,0 +1,17 @@
+package com.example.myshoppingapp.model;
+
+import lombok.Data;
+
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.MappedSuperclass;
+
+@MappedSuperclass
+@Data
+public class BaseEntity {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Integer id;
```

Comment:
de ce ai modicicat totul in Integer ?

Reply:
Nu voia sa mearga una dintre metode cu Long, se inrosea tot, daca devine totul functional si e ok,schimbam la final inapoi in Long ca nu e problema momentan,zic eu